Public/Src/Cache/ContentStore/Distributed/NuCache/CheckpointManager.cs

```diff
@@ -128,37 +131,16 @@ private async Task CreateFullCheckpointAsync(OperationContext context, EventSequ
             await _checkpointRegistry.RegisterCheckpointAsync(context, checkpointId, sequencePoint).ThrowIfFailure();
         }
 
-        private async Task<bool> CreateCheckpointIncrementalAsync(OperationContext context, EventSequencePoint sequencePoint, bool successfullyUpdatedIncrementalState)
+        private async Task CreateCheckpointIncrementalAsync(OperationContext context, EventSequencePoint sequencePoint)

```

Comment:
Logging with timings of steps?

Reply:
Each step is already wrapped in it's own performoperation.
So I don't think we need to do this explicitely once again.

---
In reply to: [319297670](https://github.com/microsoft/BuildXL/pull/818#discussion_r319297670) [](ancestors = 319297670)